build/azure-pipelines/common/extract-telemetry.sh

```diff
@@ -4,7 +4,7 @@ set -e
 cd $BUILD_STAGINGDIRECTORY
 git clone https://github.com/microsoft/vscode-telemetry-extractor.git
 cd vscode-telemetry-extractor
-git checkout 4e64f3de30f8fccb58ebdc0d85c4861a135d46cf
+git checkout f538e3157c84d1bd0b239dfc5ebccac226006d58
```

Comment:
I don't know why this is saying there's a change, I think my fork is just all messed up, this can safely be ignored as I have another PR that is throwing most of this build script away. This is unrelated to the filter extensions by category.

Reply:
Ignoring this PR as mentioned.